robot/EDA/tests/browser/settings/eda_settings_course_connections_edit_mode.robot

```diff
@@ -7,22 +7,30 @@ Library         cumulusci.robotframework.PageObjects
 Suite Setup     Open Test Browser
 Suite Teardown  Capture screenshot and delete records and close browser
 
+Test Setup      Run keywords
+...             Go to EDA settings tab          Course Connections      AND
+...             Update enable cc to default
+
 *** Test Cases ***
 Validate Edit Mode For Course Connections, Settings
```

Comment:
please add documentation to the test case

Reply:
Done